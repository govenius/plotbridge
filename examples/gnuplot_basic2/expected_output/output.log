This is pdfTeX, Version 3.14159265-2.6-1.40.16 (TeX Live 2015/Debian) (preloaded format=pdflatex 2017.8.31)  8 APR 2018 12:16
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**output
(./output.tex
LaTeX2e <2016/02/01>
Babel <3.9q> and hyphenation patterns for 29 language(s) loaded.
(/usr/share/texlive/texmf-dist/tex/latex/base/minimal.cls
Document Class: minimal 2001/05/25 Standard LaTeX minimal class
) (/usr/share/texlive/texmf-dist/tex/latex/base/size11.clo
File: size11.clo 2014/09/29 v1.4h Standard LaTeX file (size option)
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2014/10/28 v1.0g Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2016/01/03 v1.0q Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2016/01/03 v1.10 sin cos tan (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/graphics.cfg
File: graphics.cfg 2010/04/23 v1.9 graphics configuration of TeX Live
)
Package graphics Info: Driver file: pdftex.def on input line 95.

(/usr/share/texlive/texmf-dist/tex/latex/pdftex-def/pdftex.def
File: pdftex.def 2011/05/27 v0.06d Graphics/color for pdfTeX

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/infwarerr.sty
Package: infwarerr 2010/04/08 v1.3 Providing info/warning/error messages (HO)
)
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ltxcmds.sty
Package: ltxcmds 2011/11/09 v1.22 LaTeX kernel commands for general use (HO)
)
\Gread@gobject=\count79
))
\Gin@req@height=\dimen102
\Gin@req@width=\dimen103
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty
Package: color 2016/01/03 v1.1b Standard LaTeX Color (DPC)

(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/color.cfg
File: color.cfg 2007/01/18 v1.5 color configuration of teTeX/TeXLive
)
Package color Info: Driver file: pdftex.def on input line 143.
)
(/usr/share/texlive/texmf-dist/tex/latex/geometry/geometry.sty
Package: geometry 2010/09/12 v5.6 Page Geometry

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifpdf.sty
Package: ifpdf 2011/01/30 v2.3 Provides the ifpdf switch (HO)
Package ifpdf Info: pdfTeX in PDF mode is detected.
)
(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifvtex.sty
Package: ifvtex 2010/03/01 v1.5 Detect VTeX and its facilities (HO)
Package ifvtex Info: VTeX not detected.
)
(/usr/share/texlive/texmf-dist/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2010/09/12 v0.6 Provides ifxetex conditional
)
\Gm@cnth=\count80
\Gm@cntv=\count81
\c@Gm@tempcnt=\count82
\Gm@bindingoffset=\dimen104
\Gm@wd@mp=\dimen105
\Gm@odd@mp=\dimen106
\Gm@even@mp=\dimen107
\Gm@layoutwidth=\dimen108
\Gm@layoutheight=\dimen109
\Gm@layouthoffset=\dimen110
\Gm@layoutvoffset=\dimen111
\Gm@dimlist=\toks15
)
Using configuration file gnuplot.cfg
(/usr/share/texmf/tex/latex/gnuplot/gnuplot.cfg
(/usr/share/texlive/texmf-dist/tex/latex/base/fontenc.sty
Package: fontenc 2005/09/27 v1.99g Standard LaTeX package

(/usr/share/texlive/texmf-dist/tex/latex/base/t1enc.def
File: t1enc.def 2005/09/27 v1.99g Standard LaTeX file
LaTeX Font Info:    Redeclaring font encoding T1 on input line 48.
)))
No file output.aux.
\openout1 = `output.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 59.
LaTeX Font Info:    ... okay on input line 59.
LaTeX Font Info:    Try loading font information for T1+Times on input line 59.

LaTeX Font Info:    No file T1Times.fd. on input line 59.

LaTeX Font Warning: Font shape `T1/Times/m/n' undefined
(Font)              using `T1/cmr/m/n' instead on input line 59.

(/usr/share/texlive/texmf-dist/tex/context/base/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count83
\scratchdimen=\dimen112
\scratchbox=\box26
\nofMPsegments=\count84
\nofMParguments=\count85
\everyMPshowfont=\toks16
\MPscratchCnt=\count86
\MPscratchDim=\dimen113
\MPnumerator=\count87
\makeMPintoPDFobject=\count88
\everyMPtoPDFconversion=\toks17
) (/usr/share/texlive/texmf-dist/tex/generic/oberdiek/pdftexcmds.sty
Package: pdftexcmds 2011/11/29 v0.20 Utility functions of pdfTeX for LuaTeX (HO
)

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/ifluatex.sty
Package: ifluatex 2010/03/01 v1.3 Provides the ifluatex switch (HO)
Package ifluatex Info: LuaTeX not detected.
)
Package pdftexcmds Info: LuaTeX not detected.
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
)
(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/epstopdf-base.sty
Package: epstopdf-base 2010/02/09 v2.5 Base part for package epstopdf

(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/grfext.sty
Package: grfext 2010/08/19 v1.1 Manage graphics extensions (HO)

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/kvdefinekeys.sty
Package: kvdefinekeys 2011/04/07 v1.3 Define keys (HO)
))
(/usr/share/texlive/texmf-dist/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2011/06/30 v3.11 Key value format for package options (HO)

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/kvsetkeys.sty
Package: kvsetkeys 2012/04/25 v1.16 Key value parser (HO)

(/usr/share/texlive/texmf-dist/tex/generic/oberdiek/etexcmds.sty
Package: etexcmds 2011/02/16 v1.5 Avoid name clashes with e-TeX commands (HO)
Package etexcmds Info: Could not find \expanded.
(etexcmds)             That can mean that you are not using pdfTeX 1.50 or
(etexcmds)             that some package has redefined \expanded.
(etexcmds)             In the latter case, load this package earlier.
)))
Package grfext Info: Graphics extension search list:
(grfext)             [.png,.pdf,.jpg,.mps,.jpeg,.jbig2,.jb2,.PNG,.PDF,.JPG,.JPE
G,.JBIG2,.JB2,.eps]
(grfext)             \AppendGraphicsExtensions on input line 452.

(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
))
*geometry* detected driver: pdftex
*geometry* verbose mode - [ preamble ] result:
* driver: pdftex
* paper: custom
* layout: <same size as paper>
* layoutoffset:(h,v)=(0.0pt,0.0pt)
* modes: 
* h-part:(L,W,R)=(0.0pt, 602.25pt, 0.0pt)
* v-part:(T,H,B)=(0.0pt, 301.125pt, 0.0pt)
* \paperwidth=602.25pt
* \paperheight=301.125pt
* \textwidth=602.25pt
* \textheight=301.125pt
* \oddsidemargin=-72.26999pt
* \evensidemargin=-72.26999pt
* \topmargin=-109.26999pt
* \headheight=12.0pt
* \headsep=25.0pt
* \topskip=11.0pt
* \footskip=30.0pt
* \marginparwidth=4.0pt
* \marginparsep=10.0pt
* \columnsep=0.0pt
* \skip\footins=10.0pt plus 4.0pt minus 2.0pt
* \hoffset=0.0pt
* \voffset=0.0pt
* \mag=1000
* \@twocolumnfalse
* \@twosidefalse
* \@mparswitchfalse
* \@reversemarginfalse
* (1in=72.27pt=25.4mm, 1cm=28.453pt)

\gptboxheight=\skip41
\gptboxwidth=\skip42
\gptboxtext=\box27
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <10.95> on input line 208.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <8> on input line 208.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <6> on input line 208.
Package epstopdf Info: Source file: <output-inc.eps>
(epstopdf)                    date: 2018-04-08 12:16:34
(epstopdf)                    size: 24379 bytes
(epstopdf)             Output file: <output-inc-eps-converted-to.pdf>
(epstopdf)             Command: <repstopdf --outfile=output-inc-eps-converted-t
o.pdf output-inc.eps>
(epstopdf)             \includegraphics on input line 209.
runsystem(repstopdf --outfile=output-inc-eps-converted-to.pdf output-inc.eps)..
.executed safely (allowed).

Package epstopdf Info: Result file: <output-inc-eps-converted-to.pdf>
(epstopdf)                    date: 2018-04-08 12:16:36
(epstopdf)                    size: 8616 bytes.
<output-inc-eps-converted-to.pdf, id=1, 602.25pt x 301.125pt>
File: output-inc-eps-converted-to.pdf Graphic file (type pdf)

<use output-inc-eps-converted-to.pdf>
Package pdftex.def Info: output-inc-eps-converted-to.pdf used on input line 209
.
(pdftex.def)             Requested size: 602.24852pt x 301.12425pt.
 [1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map} <./output-inc-eps-conver
ted-to.pdf>] (./output.aux)

LaTeX Font Warning: Some font shapes were not available, defaults substituted.

 ) 
Here is how much of TeX's memory you used:
 2099 strings out of 494374
 29379 string characters out of 6164954
 90649 words of memory out of 5000000
 5398 multiletter control sequences out of 15000+600000
 7148 words of font info for 24 fonts, out of 8000000 for 9000
 350 hyphenation exceptions out of 8191
 38i,7n,31p,232b,243s stack positions out of 5000i,500n,10000p,200000b,80000s
{/usr/share/texmf/fonts/enc/dvips/cm-super/cm-super-t1.enc}</usr/share/texliv
e/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi10.pfb></usr/share/texlive/texm
f-dist/fonts/type1/public/amsfonts/cm/cmmi8.pfb></usr/share/texlive/texmf-dist/
fonts/type1/public/amsfonts/cm/cmr10.pfb></usr/share/texlive/texmf-dist/fonts/t
ype1/public/amsfonts/cm/cmsy10.pfb></usr/share/texmf/fonts/type1/public/cm-supe
r/sfrm1095.pfb>
Output written on output.pdf (1 page, 54379 bytes).
PDF statistics:
 32 PDF objects out of 1000 (max. 8388607)
 22 compressed objects within 1 object stream
 0 named destinations out of 1000 (max. 500000)
 6 words of extra memory for PDF output out of 10000 (max. 10000000)

