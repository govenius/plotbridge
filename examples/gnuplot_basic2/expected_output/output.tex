% GNUPLOT: LaTeX picture with Postscript
\documentclass{minimal}
% Set font size
\makeatletter
\def\@ptsize{1}
\InputIfFileExists{size11.clo}{}{%
   \GenericError{(gnuplot) \space\space\space\@spaces}{%
      Gnuplot Error: File `size11.clo' not found! Could not set font size%
   }{See the gnuplot documentation for explanation.%
   }{For using a font size a file `size<fontsize>.clo' has to exist.
        Falling back ^^Jto default fontsize 10pt.}%
  \def\@ptsize{0}
  \input{size10.clo}%
}%
\makeatother
\renewcommand*\rmdefault{Times}%
% Load packages
\usepackage{graphicx}
\usepackage{color}
\makeatletter
% Select an appropriate default driver (from TeXLive graphics.cfg)
\begingroup
  \chardef\x=0 %
  % check pdfTeX
  \@ifundefined{pdfoutput}{}{%
    \ifcase\pdfoutput
    \else
      \chardef\x=1 %
    \fi
  }%
  % check VTeX
  \@ifundefined{OpMode}{}{%
    \chardef\x=2 %
  }%
\expandafter\endgroup
\ifcase\x
  % default case
  \PassOptionsToPackage{dvips}{geometry}
\or
  % pdfTeX is running in pdf mode
  \PassOptionsToPackage{pdftex}{geometry}
\else
  % VTeX is running
  \PassOptionsToPackage{vtex}{geometry}
\fi
\makeatother
% Set papersize
\usepackage[papersize={600.00bp,300.00bp},text={600.00bp,300.00bp}]{geometry}
% No page numbers and no paragraph indentation
\pagestyle{empty}
\setlength{\parindent}{0bp}%
% Load configuration file
\InputIfFileExists{gnuplot.cfg}{%
  \typeout{Using configuration file gnuplot.cfg}%
}{%
 \typeout{No configuration file gnuplot.cfg found.}%
}%
\newcommand{\ft}[0]{\footnotesize}
\begin{document}
\begingroup
  \makeatletter
  \providecommand\color[2][]{%
    \GenericError{(gnuplot) \space\space\space\@spaces}{%
      Package color not loaded in conjunction with
      terminal option `colourtext'%
    }{See the gnuplot documentation for explanation.%
    }{Either use 'blacktext' in gnuplot or load the package
      color.sty in LaTeX.}%
    \renewcommand\color[2][]{}%
  }%
  \providecommand\includegraphics[2][]{%
    \GenericError{(gnuplot) \space\space\space\@spaces}{%
      Package graphicx or graphics not loaded%
    }{See the gnuplot documentation for explanation.%
    }{The gnuplot epslatex terminal needs graphicx.sty or graphics.sty.}%
    \renewcommand\includegraphics[2][]{}%
  }%
  \providecommand\rotatebox[2]{#2}%
  \@ifundefined{ifGPcolor}{%
    \newif\ifGPcolor
    \GPcolortrue
  }{}%
  \@ifundefined{ifGPblacktext}{%
    \newif\ifGPblacktext
    \GPblacktextfalse
  }{}%
  % define a \g@addto@macro without @ in the name:
  \let\gplgaddtomacro\g@addto@macro
  % define empty templates for all commands taking text:
  \gdef\gplbacktext{}%
  \gdef\gplfronttext{}%
  \makeatother
  \ifGPblacktext
    % no textcolor at all
    \def\colorrgb#1{}%
    \def\colorgray#1{}%
  \else
    % gray or color?
    \ifGPcolor
      \def\colorrgb#1{\color[rgb]{#1}}%
      \def\colorgray#1{\color[gray]{#1}}%
      \expandafter\def\csname LTw\endcsname{\color{white}}%
      \expandafter\def\csname LTb\endcsname{\color{black}}%
      \expandafter\def\csname LTa\endcsname{\color{black}}%
      \expandafter\def\csname LT0\endcsname{\color[rgb]{1,0,0}}%
      \expandafter\def\csname LT1\endcsname{\color[rgb]{0,1,0}}%
      \expandafter\def\csname LT2\endcsname{\color[rgb]{0,0,1}}%
      \expandafter\def\csname LT3\endcsname{\color[rgb]{1,0,1}}%
      \expandafter\def\csname LT4\endcsname{\color[rgb]{0,1,1}}%
      \expandafter\def\csname LT5\endcsname{\color[rgb]{1,1,0}}%
      \expandafter\def\csname LT6\endcsname{\color[rgb]{0,0,0}}%
      \expandafter\def\csname LT7\endcsname{\color[rgb]{1,0.3,0}}%
      \expandafter\def\csname LT8\endcsname{\color[rgb]{0.5,0.5,0.5}}%
    \else
      % gray
      \def\colorrgb#1{\color{black}}%
      \def\colorgray#1{\color[gray]{#1}}%
      \expandafter\def\csname LTw\endcsname{\color{white}}%
      \expandafter\def\csname LTb\endcsname{\color{black}}%
      \expandafter\def\csname LTa\endcsname{\color{black}}%
      \expandafter\def\csname LT0\endcsname{\color{black}}%
      \expandafter\def\csname LT1\endcsname{\color{black}}%
      \expandafter\def\csname LT2\endcsname{\color{black}}%
      \expandafter\def\csname LT3\endcsname{\color{black}}%
      \expandafter\def\csname LT4\endcsname{\color{black}}%
      \expandafter\def\csname LT5\endcsname{\color{black}}%
      \expandafter\def\csname LT6\endcsname{\color{black}}%
      \expandafter\def\csname LT7\endcsname{\color{black}}%
      \expandafter\def\csname LT8\endcsname{\color{black}}%
    \fi
  \fi
    \setlength{\unitlength}{0.0500bp}%
    \ifx\gptboxheight\undefined%
      \newlength{\gptboxheight}%
      \newlength{\gptboxwidth}%
      \newsavebox{\gptboxtext}%
    \fi%
    \setlength{\fboxrule}{0.5pt}%
    \setlength{\fboxsep}{1pt}%
\begin{picture}(12000.00,6000.00)%
    \gplgaddtomacro\gplbacktext{%
      \csname LTb\endcsname%
      \put(682,704){\makebox(0,0)[r]{\strut{}$-4$}}%
      \csname LTb\endcsname%
      \put(682,1333){\makebox(0,0)[r]{\strut{}$-3$}}%
      \csname LTb\endcsname%
      \put(682,1962){\makebox(0,0)[r]{\strut{}$-2$}}%
      \csname LTb\endcsname%
      \put(682,2591){\makebox(0,0)[r]{\strut{}$-1$}}%
      \csname LTb\endcsname%
      \put(682,3220){\makebox(0,0)[r]{\strut{}$0$}}%
      \csname LTb\endcsname%
      \put(682,3848){\makebox(0,0)[r]{\strut{}$1$}}%
      \csname LTb\endcsname%
      \put(682,4477){\makebox(0,0)[r]{\strut{}$2$}}%
      \csname LTb\endcsname%
      \put(682,5106){\makebox(0,0)[r]{\strut{}$3$}}%
      \csname LTb\endcsname%
      \put(682,5735){\makebox(0,0)[r]{\strut{}$4$}}%
      \csname LTb\endcsname%
      \put(814,484){\makebox(0,0){\strut{}$-6$}}%
      \csname LTb\endcsname%
      \put(2147,484){\makebox(0,0){\strut{}$-4$}}%
      \csname LTb\endcsname%
      \put(3480,484){\makebox(0,0){\strut{}$-2$}}%
      \csname LTb\endcsname%
      \put(4813,484){\makebox(0,0){\strut{}$0$}}%
      \csname LTb\endcsname%
      \put(6146,484){\makebox(0,0){\strut{}$2$}}%
      \csname LTb\endcsname%
      \put(7479,484){\makebox(0,0){\strut{}$4$}}%
      \csname LTb\endcsname%
      \put(8812,484){\makebox(0,0){\strut{}$6$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,704){\makebox(0,0)[l]{\strut{}$-1$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,1263){\makebox(0,0)[l]{\strut{}$-0.5$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,1822){\makebox(0,0)[l]{\strut{}$0$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,2381){\makebox(0,0)[l]{\strut{}$0.5$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,2940){\makebox(0,0)[l]{\strut{}$1$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,3499){\makebox(0,0)[l]{\strut{}$1.5$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,4058){\makebox(0,0)[l]{\strut{}$2$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,4617){\makebox(0,0)[l]{\strut{}$2.5$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,5176){\makebox(0,0)[l]{\strut{}$3$}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(8944,5735){\makebox(0,0)[l]{\strut{}$3.5$}}%
    }%
    \gplgaddtomacro\gplfronttext{%
      \csname LTb\endcsname%
      \put(176,3219){\rotatebox{-270}{\makebox(0,0){\strut{}voltage ($\mu$V)}}}%
      \colorrgb{0.00,0.00,0.00}%
      \put(9713,3219){\rotatebox{-270}{\makebox(0,0){\strut{}$\frac {dV}{dI}$ ($k\Omega$)}}}%
      \csname LTb\endcsname%
      \put(4813,154){\makebox(0,0){\strut{}current (nA)}}%
      \put(4813,5625){\makebox(0,0){\strut{}}}%
      \csname LTb\endcsname%
      \put(11012,1034){\makebox(0,0)[r]{\strut{}voltage}}%
      \csname LTb\endcsname%
      \put(11012,814){\makebox(0,0)[r]{\strut{}$\frac {dV}{dI}$}}%
    }%
    \gplbacktext
    \put(0,0){\includegraphics{output-inc}}%
    \gplfronttext
  \end{picture}%
\endgroup
\end{document}
